Java-tests fail when using annotated test-methods

When modifying `SimpleTypesTest` by using the _JUnit4-approach_ (test-method annotated with `@Test`) the consumer never receives a result:
```
java.lang.AssertionError: Eventually executed 99 times and failed 99 times after 10000 milliseconds.

	at org.junit.Assert.fail(Assert.java:88)
	at com.gruchalski.kafka.test.SimpleTypesTest$1.accept(SimpleTypesTest.java:96)
	at com.gruchalski.kafka.test.SimpleTypesTest$1.accept(SimpleTypesTest.java:44)
	at java.util.Optional.ifPresent(Optional.java:159)
	at com.gruchalski.kafka.test.SimpleTypesTest.testClusterSetup(SimpleTypesTest.java:44)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:68)
	at com.intellij.rt.execution.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:51)
	at com.intellij.rt.execution.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:237)
	at com.intellij.rt.execution.junit.JUnitStarter.main(JUnitStarter.java:70)
```

--- a/src/test/java/com/gruchalski/kafka/test/SimpleTypesTest.java
+++ b/src/test/java/com/gruchalski/kafka/test/SimpleTypesTest.java
@@ -23,7 +23,8 @@ import com.gruchalski.kafka.scala.ConsumedItem;
 import com.gruchalski.kafka.scala.KafkaTopicConfiguration;
 import com.gruchalski.kafka.scala.KafkaTopicCreateResult;
 import com.gruchalski.testing.AsyncUtil;
-import junit.framework.TestCase;
+import static org.junit.Assert.*;
+import org.junit.Test;
 
 import java.util.ArrayList;
 import java.util.List;
@@ -34,9 +35,10 @@ import java.util.concurrent.CountDownLatch;
 import java.util.concurrent.TimeUnit;
 import java.util.function.Consumer;
 
-public class SimpleTypesTest extends TestCase {
+public class SimpleTypesTest {
     private String aKey = "a test key";
     private String aValue = "a test value";
+    @Test
     public void testClusterSetup() {
         new KafkaCluster().start().ifPresent(new Consumer<KafkaClusterSafe>() {
             @Override
@@ -83,7 +85,7 @@ public class SimpleTypesTest extends TestCase {
                             Optional<ConsumedItem<String, String>> consumed = kafkaClusterSafe.cluster.consume(
                                     topics.get(0).name(), String.class, String.class);
 
-                            assertEquals(consumed.get().key(), aKey);
+                            assertEquals(consumed.get().key().get(), aKey);
                             assertEquals(consumed.get().value(), aValue);
                         } catch (Throwable t) {
                             fail(t.getMessage());

